mixer/pkg/server/server.go

```diff
@@ -143,16 +143,21 @@ func newServer(a *Args, p *patchTable) (*Server, error) {
 		return nil, fmt.Errorf("unable to listen on socket: %v", err)
 	}
 
-	configStoreURL := a.ConfigStoreURL
-	if configStoreURL == "" {
-		configStoreURL = "k8s://"
-	}
+	store2 := a.ConfigStore
+	if store2 != nil && a.ConfigStoreURL != "" {
+		return nil, fmt.Errorf("invalid arguments: both ConfigStore and ConfigStoreURL are specified")
+	}
+	if store2 == nil {
```

Comment:
Could you get rid of all the "2" suffixes in this file? They're leftovers from the past...

Reply:
done -- actually, I couldn't rename this to `store` since it conflicts with the package name itself used below -- and `s` is already used for the server. I chose `st` but please tell me if you have some better ideas.